src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write code here
+function convertToNum(num) {
+  return num.split(',').join('').split('$').join('');
+};
+
+function sortBySalary(list) {
+  const sorted = [...list].sort(function(a, b) {
+    const FirstSalary = convertToNum(a.dataset.salary);
```

Comment:
use camelCase for variable naming.

[More info](https://ru.wikipedia.org/wiki/CamelCase).